tests/unit/config/test_runtime.py

```diff
@@ -16,7 +18,15 @@
 )
 
 
-class TestRuntimeConfig(BaseConfigTest):
+class TestRuntimeConfig:
+    def test_str(self, profile: Profile, project: Project) -> None:
+        config = dbt.config.RuntimeConfig.from_parts(project, profile, {})
+
+        # to make sure nothing terrible happens
+        str(config)
+
+
+class TestRuntimeConfigOLD(BaseConfigTest):
```

Comment:
This class, `TestRuntimeConfigOLD`, goes away in a later commit (https://github.com/dbt-labs/dbt-core/pull/10242/commits/ce46f02340f84b439dfee10e98d66ef0a26bb7b0) after we've moved all the tests it contains.